functions/dataManger/priceDataManager.ts

```diff
@@ -0,0 +1,63 @@
+import { Game } from "../models/game";
```

Comment:
Since this file is completely priceCharting specific you might want to name it as such and pull out the common code into its own class or interface. .